api/filter/http/health_check.proto

```diff
@@ -0,0 +1,21 @@
+syntax = "proto3";
+
+package envoy.api.v2.filter.http;
+
+import "google/protobuf/duration.proto";
+import "google/protobuf/wrappers.proto";
+
+// Note that the filter will automatically fail health checks and set the
+// x-envoy-immediate-health-check-fail header if the /healthcheck/fail
+// admin endpoint has been called. (The /healthcheck/ok admin endpoint
+// reverses this behavior).
+message HealthCheck {
+  // Specifies whether the filter operates in pass through mode or not. REQUIRED.
+  google.protobuf.BoolValue pass_through_mode = 1;
+  // Specifies the incoming HTTP endpoint that should be considered the
+  // health check endpoint. For example /healthcheck.
```

Comment:
required? if not what is default?

Reply:
This was @htuch’s suggestion. The required term is only for internal tracking, when we have wrapper types and the field is required. It’s not a doc level indicator of required fields. I can do that but better done in separate PR as one full sweep, with no proto change.

Reply:
OK that's fine we can fix this in a follow up.